node/service/src/lib.rs

```diff
@@ -33,19 +33,20 @@ use cumulus_primitives_core::ParaId;
 use cumulus_primitives_parachain_inherent::{MockValidationDataInherentDataProvider, MockXcmConfig};
 use cumulus_relay_chain_inprocess_interface::build_inprocess_relay_chain;
 use cumulus_relay_chain_interface::{RelayChainError, RelayChainInterface, RelayChainResult};
-use cumulus_relay_chain_rpc_interface::RelayChainRPCInterface;
+use cumulus_relay_chain_rpc_interface::{create_client_and_start_worker, RelayChainRpcInterface};
 pub use futures::stream::StreamExt;
 use jsonrpsee::RpcModule;
 use sc_consensus::LongestChain;
 use sc_consensus_aura::{ImportQueueParams, StartAuraParams};
 use sc_executor::WasmExecutor;
 use sc_network::NetworkService;
+use sc_network_common::service::NetworkBlock;
```

Comment:
@zjb0807 could you please explain why this import is required here? and also in general when to use stuffs from sc-network-common?

Reply:
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/16496415/213362319-56496b99-fabd-44c0-a459-c8b76b2d6ea2.png">

Remove and rebuild, you'll see some errors and suggestions